playground/index.html

```diff
@@ -3,11 +3,29 @@
 <head>
     <meta charset="UTF-8">
     <title>Scratch WebGL rendering demo</title>
+    <style>
+        #scratch-stage { width: 480px; }
+        #debug-canvas { width: 480px; }
+    </style>
 </head>
 <body style="background: lightsteelblue">
 <canvas id="debug-canvas" width="10" height="10" style="border:3px dashed red"></canvas>
 <canvas id="scratch-stage" width="10" height="10" style="border:3px dashed black"></canvas>
 <p>
+    <select id="fudgeproperty" onchange="onFudgePropertyChanged(this.value)">
```

Comment:
I also added this to the playground to let us play with multiple properties at once more easily....